src/kademlia/dht_tracker.cpp

```diff
@@ -192,6 +235,9 @@ namespace libtorrent { namespace dht
 		if (e || m_abort) return;
 
 		m_dht.tick();
+#if TORRENT_USE_IPV6
+		m_dht6.tick();
```

Comment:
The problem is that dht::node is neither CopyConstructible nor DefaultConstructible so it cannot be initialized via aggregate initialization in C++03. This would require C++11.